gateway/api/use_cases/authentication.py

```diff
@@ -40,13 +40,13 @@ def __init__(
         self.public_access = public_access
 
     def _get_authentication_service_instance(self) -> AuthenticationBase:
-        if self.channel == Channel.IBM_CLOUD:
+        if self.channel == Channel.IBM_QUANTUM_PLATFORM:
```

Comment:
With this change, would authentication with "ibm_cloud" still work? I haven't had the chance to test it but it looks like it might fail (unless I am missing something being set to iqp under the hood)

Reply:
Yes, this is what I was commenting you the other day. We basically don't send the channel if not that if you send us an instance we assume you will use the `ibm_cloud` / `ibm_quantum_platform`: [ref](https://github.com/Qiskit/qiskit-serverless/blob/main/gateway/api/authentication.py#L43).

The idea is, internally, we just don't make use of channels. Thinking that, in the near future, how we are going to have only one channel we are going to be able to remove this logic.